src/main/java/mat/server/CqlValidatorRemoteCallService.java

```diff
@@ -0,0 +1,40 @@
+package mat.server;
+
+import mat.client.shared.MatRuntimeException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+@Service
+public class CqlValidatorRemoteCallService {
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    @Value("${CQL_ELM_TRANSLATION_URL:http://localhost:7070}")
+    private String cqlElmtranslationUrl;
+
+    private String CQL_VALIDATION_RESOURCE = "/cql/translator/cql";
+
+    public String validateCqlExpression(String cqlFileString) {
+
```

Comment:
Was it possible to map a POJO response object with Jason instead of String?

Reply:
Other than that it looks OK to me.
there is a codacity waring

Reply:
I am looking into if I could map it to POJO.